src/spyder_updater/scripts/install.sh

```diff
@@ -1,15 +1,25 @@
 #!/bin/bash
 
-while getopts "i:c:p:r" option; do
+while getopts "i:c:p:rs" option; do
```

Comment:
```suggestion
while getopts "i:c:p:r:s" option; do
```

Is this not missing? I don't know how `getopts` works but it seems to be according to the current options.

Reply:
It is not missing. the `:` means that there is a value argument to read. `-i` should be followed by a file, `-c` should be followed by the conda path, `-p` should be followed by the environment prefix, and `-r` and `-s` do not have values to read.